_includes/header.html

```diff
@@ -4,7 +4,7 @@
   <ul class="platform-header__links">
     <li><a href="https://zapier.com/platform/how-to-build">How to Build</a></li>
     <li><a href="https://zapier.com/platform/partner-program">Partner Program</a></li>
-    <li><a href="https://platform.zapier.com/">Documentation</a></li>
+    <li><a href="/" class="platform-header__selected">Documentation</a></li>
```

Comment:
Also brought back `href="/"` as it helps with testing locally (it goes to localhost for example instead of the production site)